pkg/services/ngalert/image/service.go

```diff
@@ -186,7 +207,9 @@ func (s *ScreenshotImageService) NewImage(ctx context.Context, r *models.AlertRu
 
 	image := result.(models.Image)
 	if err = s.cache.Set(ctx, r.GetKey().String(), image); err != nil {
-		s.logger.Warn("Failed to cache image", "token", image.Token, "error", err)
+		s.logger.Warn("Failed to cache image",
+			"token", image.Token,
+			"error", err)
```

Comment:
the context in log above is pretty much the same. I wonder if it is better to just declare a logger once with all context we need and then use it.